__tests__/app.test.js

```diff
@@ -191,3 +191,39 @@ describe("GET /api/articles/:article_id", () => {
       })
   })
 })
+describe.only('GET api/articles', () => {
+  test('respond with an array with articles object', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({ body }) => {
+      const { articles } = body
+      articles.forEach((article) => {
```

Comment:
If you are using a forEach, you want to also test for the expected length of articles array to make sure that it is running for all the objects which would be more robust.

Reply:
done